spec/models/npq_course_spec.rb

```diff
@@ -4,13 +4,15 @@
 RSpec.describe NPQCourse do
   describe "::schedule_for", :with_default_schedules do
     let(:npq_course) { build(:npq_course, identifier:) }
+    let(:cohort) { Cohort.previous }
```

Comment:
I was wondering why we're using previous and not defaulting to current?

Reply:
this should be current, fixed now